<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>AdvSearchBase</class>
  <widget class="QDialog" name="AdvSearchBase">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>545</width>
        <height>537</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Advanced search</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>2</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabel2">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                    <enum>QFrame::Plain</enum>
                  </property>
                  <property name="margin">
                    <number>4</number>
                  </property>
                  <property name="text">
                    <string>Search for &lt;br&gt;documents&lt;br&gt;satisfying:</string>
                  </property>
                  <property name="toolTip" stdset="0">
                    <string>All non empty fields on the right will be combined with AND (&quot;All clauses&quot; choice) or OR (&quot;Any clause&quot; choice) conjunctions. &lt;br&gt;&quot;Any&quot; &quot;All&quot; and &quot;None&quot; field types can accept a mix of simple words, and phrases enclosed in double quotes.&lt;br&gt;Fields with no data are ignored.</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QComboBox" name="conjunctCMB">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>8</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="toolTip" stdset="0">
                    <string>All non empty fields on the right will be combined with AND (&quot;All clauses&quot; choice) or OR (&quot;Any clause&quot; choice) conjunctions. &lt;br&gt;&quot;Any&quot; &quot;All&quot; and &quot;None&quot; field types can accept a mix of simple words, and phrases enclosed in double quotes.&lt;br&gt;Fields with no data are ignored.</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2">
                  <property name="sizeHint">
                    <size>
                      <width>0</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QPushButton" name="delClausePB">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Delete clause</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="addClausePB">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Add clause</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="Q3Frame" name="clauseFRM">
              <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
                <enum>QFrame::Plain</enum>
              </property>
              <layout class="QVBoxLayout">
                <property name="margin">
                  <number>0</number>
                </property>
                <property name="spacing">
                  <number>1</number>
                </property>
                <item>
                  <widget class="Line" name="clauseline">
                    <property name="frameShape">
                      <enum>QFrame::HLine</enum>
                    </property>
                    <property name="frameShadow">
                      <enum>QFrame::Plain</enum>
                    </property>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="line4">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QCheckBox" name="restrictFtCB">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Restrict file types</string>
              </property>
              <property name="toolTip" stdset="0">
                <string>Check this to enable filtering on file types</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QCheckBox" name="restrictCtCB">
              <property name="enabled">
                <bool>false</bool>
              </property>
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>1</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>1</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>By categories</string>
              </property>
              <property name="toolTip" stdset="0">
                <string>Check this to use file categories instead of raw mime types</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="saveFileTypesPB">
              <property name="enabled">
                <bool>false</bool>
              </property>
              <property name="text">
                <string>Save as default</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabel2_2">
                  <property name="text">
                    <string>Searched file types</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Q3ListBox" name="yesFiltypsLB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>200</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="selectionMode">
                    <enum>Q3ListBox::Extended</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>0</number>
              </property>
              <item>
                <widget class="QPushButton" name="delAFiltypPB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string>All ----&gt;</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="delFiltypPB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string>Sel -----&gt;</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="addFiltypPB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string>&lt;----- Sel</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="addAFiltypPB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="text">
                    <string>&lt;----- All</string>
                  </property>
                  <property name="autoDefault">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="spacing">
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabel3">
                  <property name="text">
                    <string>Ignored file types</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="Q3ListBox" name="noFiltypsLB">
                  <property name="enabled">
                    <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>200</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="selectionMode">
                    <enum>Q3ListBox::Extended</enum>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="line5">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QGridLayout">
          <item row="1" column="0">
            <widget class="QComboBox" name="subtreeCMB">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>7</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>8</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>300</width>
                  <height>0</height>
                </size>
              </property>
              <property name="editable">
                <bool>true</bool>
              </property>
              <property name="maxCount">
                <number>20</number>
              </property>
              <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
              </property>
              <property name="duplicatesEnabled">
                <bool>false</bool>
              </property>
              <property name="toolTip" stdset="0">
                <string>Enter top directory for search</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QPushButton" name="browsePB">
              <property name="text">
                <string>Browse</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="textLabel1">
              <property name="text">
                <string>Restrict results to files in subtree:</string>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="line1">
          <property name="frameShape">
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <widget class="QPushButton" name="searchPB">
              <property name="text">
                <string>Start Search</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="dismissPB">
              <property name="text">
                <string>Close</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
